pkg/names/names.go

```diff
@@ -31,3 +36,37 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // MULTUS_VALIDATING_WEBHOOK is the name of the ValidatingWebhookConfiguration for multus-admission-controller
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
+
+// TRUST_BUNDLE_CONFIGMAP_KEY is the name of the data key containing
+// the PEM encoded trust bundle.
+const TRUST_BUNDLE_CONFIGMAP_KEY = "ca-bundle.crt"
+
+// TRUST_BUNDLE_CONFIGMAP is the name of the ConfigMap
+// containing the combined user/system trust bundle.
+const TRUST_BUNDLE_CONFIGMAP = "trusted-ca-bundle"
+
+// TRUST_BUNDLE_CONFIGMAP_NS is the namespace that hosts the
+// ADDL_TRUST_BUNDLE_CONFIGMAP and TRUST_BUNDLE_CONFIGMAP
+// ConfigMaps.
+const TRUST_BUNDLE_CONFIGMAP_NS = "openshift-config-managed"
+
+// TRUST_BUNDLE_CONFIGMAP_ANNOTATION is the name of the annotation that
+// determines whether or not to inject the combined ca certificate
+const TRUST_BUNDLE_CONFIGMAP_ANNOTATION = "config.openshift.io/inject-trusted-cabundle"
```

Comment:
s/TRUST_BUNDLE_CONFIGMAP_ANNOTATION/TRUSTED_CA_BUNDLE_CONFIGMAP_ANNOTATION/

Reply:
I think @deads2k is going to suggest this name include proxy for the same reasons he mentioned in another PR (that we might have other bundles to inject in the future).  but i'll let him comment.

Reply:
and @smarterclayton since we went down this path somewhat at your behest, if you do *not* think we should be anticipating multiple bundles, would also like to hear from you.

Reply:
@smarterclayton is going to weigh in on this, please don't rename anything yet.